pallets/assets/src/functions.rs

```diff
@@ -1153,4 +1160,8 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
             })
             .collect::<Vec<_>>()
     }
+    pub fn remove_asset_locks(asset_id: &T::AssetId) {
+        let _ = AssetLocks::<T, I>::clear_prefix(asset_id, 0, None);
+        let _ = Locked::<T, I>::clear_prefix(asset_id, 0, None);
+    }
```

Comment:
```suggestion
    pub fn remove_asset_locks(asset_id: &T::AssetId, limit: u32) {
        let _ = AssetLocks::<T, I>::clear_prefix(asset_id, limit, None);
        let _ = Locked::<T, I>::clear_prefix(asset_id, limit, None);
    }
```

add new pallet call to call the function